lib/mri/neo4j/driver/types/uuid.rb

```
@@ -0,0 +1,29 @@
+
+module Neo4j
+  module Driver
+    module Types
+      # A Neo4j UUID value (Bolt 6.1+). An opaque, immutable value type
+      # mirroring java.util.UUID (which JRuby uses natively) and Python's
+      # uuid.UUID — construct via `from_string`, read the canonical
+      # hyphenated form via `to_s`. `new` is private so flavor-agnostic
+      # code goes through `from_string`, the one constructor that also
+      # exists on java.util.UUID.
+      class UUID
+        def self.from_string(value) = new(value)
```

RuboCop (1.88.2)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)